.gitignore

```diff
@@ -1,4 +1,4 @@
-raylib-master
+# raylib
```

Comment:
this change confused me @ViridianGames - did you originally NOT want updates to the raylib directory tracked?